app/helpers/application_helper/button/rbac_common_feature_button.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::RbacCommonFeatureButton < ApplicationHelper::Button::GenericFeatureButton
+  delegate :rbac_common_feature_for_buttons, :to => :@view_context
+
+  def role_allows_feature?
+    role_allows?(:feature => rbac_common_feature_for_buttons(@feature))
```

Comment:
I agree that it look confusing when you have button named `RbacCommonFeatureButton` and you intentionally don't set any feature. Also I think, that it will hide button, which probably isn't right.